IPython/extensions/autoreload.py

```diff
@@ -581,8 +658,9 @@ def aimport(self, parameter_s="", stream=None):
         %aimport foo, bar
         Import modules 'foo', 'bar' and mark them to be autoreloaded for %autoreload 1
 
-        %aimport -foo
-        Mark module 'foo' to not be autoreloaded for %autoreload 1
+        %aimport -foo, bar
+        Mark module 'foo' to not be autoreloaded for %autoreload 1, 2, or 3, and 'bar'
+        to be autoreloaded for 1.
```

Comment:
Feel welcome to replace the digits with verbose names here and in the paragraph above if you agree that this is a good idea.

Reply:
Done; thanks for the suggestion.